Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Defrag - centralized API for the openSUSE Infrastructure
+# Copyright (C) 2021 openSUSE contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+#!BuildTag: defrag
+
+FROM opensuse/tumbleweed
+
+ENV ENV "True"
+EXPOSE 8000
+
+RUN zypper ar https://download.opensuse.org/repositories/home:/KaratekHD:/defrag/openSUSE_Tumbleweed/ defrag
```

Comment:
Is it correct to say that if any of the following packages:
- atoma
- fastapi
- lunr
- mmh3
- pottery

gets an update, we will need to rebuild the corresponding package from OBS? If yes, I think it's not very sustainable for two developers, is it?

Reply:
We'll, it's a matter of running `py2pack fetch atoma` and `py2pack generate atoma -f python-atoma.spec` - I can easily do this alone, I did these libs in like half an hour. 
The issue with installing from requirements.txt is that we can not automate it in OBS, meaning that we would need to rebuild the containers by hand. Since currently we have hardcoded versions in the requirements.txt anyway, I think it is fine to update the packages on OBS. I can do it so you don't have to worry ;)

Reply:
If you want to further discuss this, I'd suggest having a small meeting about it.

Reply:
The problem is that you will need to remember that any of these packages *might* have a pending update and to run the commands. This means that you are now shifting an essential part of the workflow away from the codebase to this home repo. It's WAY too early for me. I want (me / the devs)  to be able to develop with their eyes focused on the codebase, automatically get the latest update without the overhead of having to remember to update some remote repo. Again, we are conflating two concerns here:
- going to production (the current goal as per the recent meetings)
- packaging (the long-term goal which you have already made a lot of progress with)

I want to to be able to focus on the first objective, the rest can wait another PR. There is no need to lump together these two separate goals.

Reply:
Sure, we can skip packaging for now. But this would also mean not to use Podman for the deployment, as we are not able to build the containers on OBS otherwise. We can of course do that, write a simple .service file, install all the requirements from the file and run it with systemd. It's not fancy, but it is a solution.

Reply:
If you're concern is that we could forget updating these packages, I can write `_service` files for the package so that the process gets automated

Reply:
> Sure, we can skip packaging for now. But this would also mean not to use Podman for the deployment, as we are not able to build the containers on OBS otherwise. We can of course do that, write a simple .service file, install all the requirements from the file and run it with systemd. It's not fancy, but it is a solution.

Well, we don't _have_ to use OBS for now. We can simply build from podman and deploy from there.

Reply:
Well, the updates for the deployment wouldn't need to occur everytime we push to dev. We probably should merge dev into main soon, build the package from main and deploy that. That way we can keep what we already have and develop without worries in dev. I mean, I would need to do the updates on the server side anyway, so I can as we trigger a service run for OBS then.